config/locales/en.yml

```diff
@@ -804,18 +804,20 @@ en:
               hint: For example, the trainee is not eligible or is self-funded
   pages:
     request_an_account:
-      inviting_limited_itt:
-        We are only inviting a limited amount of initial teacher training providers to use Register,
-        to begin with. If your organisation has not received an invite from the DfE you cannot request an account yet.
-      to_get_setup: To get set up on Register trainee teachers you need a DfE Sign-in account.
+      part1: To get a Register trainee teachers (Register) account you need a DfE Sign-in account.
```

Comment:
really small point, but wonder if we want to name this `first_para` to be consistent with `final_para` ?